config.js

```diff
@@ -42,23 +43,27 @@ System.config({
       "aurelia-logging": "npm:aurelia-logging@1.0.0-beta.1.1.1",
       "aurelia-metadata": "npm:aurelia-metadata@1.0.0-beta.1.1.3",
       "aurelia-pal": "npm:aurelia-pal@1.0.0-beta.1.1.1",
-      "core-js": "npm:core-js@2.0.3"
+      "core-js": "npm:core-js@2.1.0"
     },
     "npm:aurelia-event-aggregator@1.0.0-beta.1.1.0": {
       "aurelia-logging": "npm:aurelia-logging@1.0.0-beta.1.1.1"
     },
     "npm:aurelia-metadata@1.0.0-beta.1.1.3": {
       "aurelia-pal": "npm:aurelia-pal@1.0.0-beta.1.1.1",
-      "core-js": "npm:core-js@2.0.3"
+      "core-js": "npm:core-js@2.1.0"
+    },
+    "npm:aurelia-pal-browser@1.0.0-beta.1.1.3": {
+      "aurelia-pal": "npm:aurelia-pal@1.0.0-beta.1.1.1",
+      "core-js": "npm:core-js@2.1.0"
     },
     "npm:aurelia-route-recognizer@1.0.0-beta.1.1.0": {
       "aurelia-path": "npm:aurelia-path@1.0.0-beta.1.1.0",
-      "core-js": "npm:core-js@2.0.3"
+      "core-js": "npm:core-js@2.1.0"
     },
     "npm:babel-runtime@5.8.35": {
       "process": "github:jspm/nodelibs-process@0.1.2"
     },
-    "npm:core-js@2.0.3": {
+    "npm:core-js@2.1.0": {
```

Comment:
Does this need to be coordinated with the version of core-js in use by other aurelia libs? Or maybe it doesn't matter if it's being removed. @EisenbergEffect